kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/VirtualKafkaClusterReconciler.java

```diff
@@ -96,6 +102,28 @@ public UpdateControl<VirtualKafkaCluster> reconcile(VirtualKafkaCluster cluster,
 
     private @NonNull UpdateControl<VirtualKafkaCluster> maybeCombineStatusWithClusterConfigMap(VirtualKafkaCluster cluster, Context<VirtualKafkaCluster> context) {
         UpdateControl<VirtualKafkaCluster> updateControl;
+
+        var existingKubernetesServices = context.getSecondaryResources(Service.class)
```

Comment:
I've shared knowledge of how the cluster-ip ingress is materialised into a service with this class.   As soon as we introduce a second ingress type (LoadBalancer), my intent to to refactor this so that this class won't need to know.   I **expect** it will be given to a refactored `IngressDefinition` - but I want to defer that for now.